### PR TITLE
UI changes

### PR DESCRIPTION
**Pull request by isabelleleeson**
image
image
image

### CODE REVIEW

File: android/app/src/main/res/drawable/border.xml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-        android:bottomRightRadius="0dp"
-        android:radius="2dp"
-        android:topRightRadius="0dp" />
+        android:radius="3dp"/>
     <stroke
         android:width="1dp"
-        android:color="@android:color/black" />
+        android:color="@color/gray" />
+    <solid
+        android:color="@color/very_light_gray" />
```
Comment (jimzhaozhengwang) [inline android/app/src/main/res/drawable/border.xml]: `reformat code`

File: android/app/src/main/res/layout/activity_parent_quest_details.xml [hunk 2]
```diff
@@ -5,73 +5,80 @@
...
+        android:padding="25dp">
 
         <ImageView
             android:id="@+id/quest_image_view"
             android:layout_width="80dp"
             android:layout_height="80dp"
             android:layout_centerHorizontal="true"
-            android:layout_marginTop="20dp" />
+            android:layout_marginBottom="20dp"
+            android:src="@drawable/broom"/>
```
Comment (jimzhaozhengwang) [inline android/app/src/main/res/layout/activity_parent_quest_details.xml]: why is `broom` hardcoded?

File: android/app/src/main/res/layout/child_recycler_view_item.xml [hunk 3]
```diff
@@ -5,53 +5,54 @@
     android:background="@drawable/border"
     android:clickable="true"
     android:focusable="true"
-    android:gravity="center">
+    android:gravity="center"
```
Comment (jimzhaozhengwang) [inline android/app/src/main/res/layout/child_recycler_view_item.xml]: `reformat code`

File: android/app/src/main/res/drawable/border.xml [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 </shape>
```
Comment (jimzhaozhengwang) [inline android/app/src/main/res/drawable/border.xml]: add a new line

File: android/app/src/main/res/layout/activity_parent_quest_details.xml [hunk 5]
```diff
@@ -5,73 +5,80 @@
...
             android:text="@string/exp"
+            android:gravity="center"
             android:textSize="20sp" />
 
         <ImageView
             android:id="@+id/quest_status_image_view"
             android:layout_width="40dp"
             android:layout_height="40dp"
-            android:layout_below="@id/quest_exp_edit_text"
+            android:layout_below="@+id/quest_exp_edit_text"
```
Comment (jimzhaozhengwang) [inline android/app/src/main/res/layout/activity_parent_quest_details.xml]: remove `+`

File: android/app/src/main/res/layout/activity_parent_quest_details.xml [hunk 6]
```diff
@@ -5,73 +5,80 @@
...
             android:layout_marginTop="10dp"
+            android:layout_marginBottom="10dp"
             android:src="@drawable/yellow_check_mark" />
 
         <TextView
             android:id="@+id/quest_status_text_view"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_below="@id/quest_exp_edit_text"
+            android:layout_below="@+id/quest_exp_edit_text"
```
Comment (jimzhaozhengwang) [inline android/app/src/main/res/layout/activity_parent_quest_details.xml]: remove `+`

File: android/app/src/main/res/layout/activity_parent_quest_details.xml [hunk 7]
```diff
@@ -5,73 +5,80 @@
...
             android:textSize="20sp" />
 
         <Button
             android:id="@+id/view_proof_button"
             android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_below="@id/quest_exp_edit_text"
-            android:layout_marginStart="280dp"
+            android:layout_height="50dp"
+            android:layout_below="@+id/quest_exp_edit_text"
```
Comment (jimzhaozhengwang) [inline android/app/src/main/res/layout/activity_parent_quest_details.xml]: remove `+`

File: android/app/src/main/res/layout/child_recycler_view_item.xml [hunk 8]
```diff
@@ -5,53 +5,54 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_toEndOf="@id/child_image_view">
+        <TextView
+            android:id="@+id/child_name_text_view"
+            android:layout_width="200dp"
+            android:layout_height="50dp"
+            android:layout_marginStart="5dp"
+            android:layout_marginTop="10dp"
+            android:layout="@id/child_image_view"
```
Comment (jimzhaozhengwang) [inline android/app/src/main/res/layout/child_recycler_view_item.xml]: ?

File: android/app/src/main/res/layout/quest_recycler_view_item.xml [hunk 9]
```diff
@@ -4,48 +4,63 @@
...
-        android:textSize="20sp" />
+        android:layout_toEndOf="@+id/quest_image_view"
+        android:textSize="20sp"/>
 
     <TextView
         android:id="@+id/quest_owner_text_view"
         android:layout_width="wrap_content"
-        android:layout_height="30dp"
+        android:layout_height="wrap_content"
+        android:layout_toEndOf="@+id/quest_image_view"
```
Comment (jimzhaozhengwang) [inline android/app/src/main/res/layout/quest_recycler_view_item.xml]: remove `+`

File: android/app/src/main/res/layout/quest_recycler_view_item.xml [hunk 10]
```diff
@@ -4,48 +4,63 @@
...
+
 
     <TextView
         android:id="@+id/quest_name_text_view"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_toEndOf="@id/quest_image_view"
         android:layout="@id/quest_image_view"
-        android:textSize="20sp" />
+        android:layout_toEndOf="@+id/quest_image_view"
```
Comment (jimzhaozhengwang) [inline android/app/src/main/res/layout/quest_recycler_view_item.xml]: remove `+`

File: android/app/src/main/res/layout/quest_recycler_view_item.xml [hunk 11]
```diff
@@ -4,48 +4,63 @@
...
+        android:layout_marginStart="15dp"
+        android:layout_marginEnd="25dp"/>
+
 
     <TextView
         android:id="@+id/quest_name_text_view"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_toEndOf="@id/quest_image_view"
         android:layout="@id/quest_image_view"
```
Comment (jimzhaozhengwang) [inline android/app/src/main/res/layout/quest_recycler_view_item.xml]: ?

File: android/app/src/main/res/layout/quest_recycler_view_item.xml [hunk 12]
```diff
@@ -4,48 +4,63 @@
...
+
+    <RelativeLayout
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_below="@id/quest_image_view"
-        android:layout_marginTop="5dp"
-        android:layout_marginStart="150dp"
-        android:textSize="17sp" />
+        android:layout_marginTop="20dp"
+        android:layout_below="@+id/quest_owner_text_view"
```
Comment (jimzhaozhengwang) [inline android/app/src/main/res/layout/quest_recycler_view_item.xml]: remove `+`

File: android/app/src/main/res/layout/activity_parent_quest_details.xml (hunk 2 above)
Comment (isabelleleeson) [reply android/app/src/main/res/layout/activity_parent_quest_details.xml]: just makes it easier to see without having to build forgot to remove it tho

File: android/app/src/main/res/layout/fragment_quests_upcoming.xml [hunk 13]
```diff
@@ -8,37 +8,37 @@
...
             android:textSize="25sp" />
 
         <androidx.recyclerview.widget.RecyclerView
-            android:id="@+id/overdue_quests_recycler_view"
+            android:id="@+id/upcoming_quests_recycler_view"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_below="@id/overdue_text_view"
+            android:layout_height="match_parent"
+            android:layout_below="@+id/upcoming_text_view"
```
Comment (jimzhaozhengwang) [inline android/app/src/main/res/layout/fragment_quests_upcoming.xml]: remove `+`

**Pull request merged**